ast2vec/repo2/base.py

```diff
@@ -224,6 +256,26 @@ def ret(name):
             if last:
                 yield from ret(last)
 
+    @staticmethod
+    def prepare_reponame(reponame: str) -> str:
+        """
+        prepare name of repository for operations with git.
+        Remove '\n', '/' and '\' in the end of string
+        Add '.git' to the end of name if necessary
+        Add 'https://' in the beginning
+        :param reponame: raw name of repository
+        :return: good ready for use name
+        """
+        bad_endings = '\n\r\\/'
+        for end in bad_endings:
+            if reponame.endswith(end):
```

Comment:
OK, this is clearly reponame.rstrip("\n\r\\/")

Please notice - we always use " instead of '